### PR TITLE
Decapitalize letters, fix typos and shorten sentences in README.md

### PR DESCRIPTION
**Pull request by Kazevic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ A Reddit client on Android written in Java. It does not have any ads and it feat
 
 <div align="center">
 
-Infinity for Reddit is available on Google Play and F-Droid
-
+Infinity for Reddit is available on Google Play Store and F-Droid.
+
```
Comment (patrickdemers6) [inline README.md:19]: Remove whitespace.

File: README.md [hunk 2]
```diff
@@ -136,24 +136,24 @@ Key features of **Infinity For Reddit**:
 
 ## Contributing
 
-First off, thanks for taking the time to contribute! Contributions are what makes the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+First off, thanks for taking the time to contribute! Contributions are what makes the open source community such an amazing place to create, inspire, and learn. Any contributions you make are **greatly appreciated**.
 
-If you have a suggestion that would make this better, please fork the repo and create a pull request.
-It's better to also open an issue describing the issue you want to fix. But it is not required.
+Pease fork the repository and open a pull request if you have a suggestion that would make this better.
```
Comment (patrickdemers6) [inline README.md:141]: Would being specific about "this" be better? Such as "the application" or "Infinity for Reddit"?

**Review by patrickdemers6 — CHANGES_REQUESTED** [review]
Great suggestions! Just a few small pieces of feedback from me.

File: README.md (hunk 2 above)
Comment (Kazevic) [reply README.md]: Now that you mention it, I think so too. Thanks, it should be good to go now!

**Pull request closed without merging**